src/libraries/System.Text.Json/gen/TypeGenerationSpec.cs

```diff
@@ -19,7 +19,7 @@ internal class TypeGenerationSpec
         public string TypeRef { get; private set; }
 
         /// <summary>
-        /// The name of the public JsonTypeInfo<T> property for this type on the generated context class.
+        /// The name of the public JsonTypeInfo&lt;T&gt; property for this type on the generated context class.
```

Comment:
@carlossanlop we can't use cref's here since the source generator doesn't reference STJ. Would it perhaps make sense to use xref's though?

Reply:
@eiriktsarpalis no, xrefs are only available in markdown. What you can do here is wrap the docID with `<c></c>`:

```suggestion
        /// The name of the public <c>JsonTypeInfo&lt;T&gt;</c> property for this type on the generated context class.
```